MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,341 @@
+        set { this.customer = value; }
+    }
+
+    public string ShipCity {
+        get { return shipCity; }
+        set { this.shipCity = value; }
+    }
+
+    public OrderInfo (string orderId, string customerId, string country, string customer, string shipCity)
+    {
+        this.OrderID = orderId;
+        this.CustomerID = customerId;
+        this.Customer = customer;
+        this.ShipCountry = country;
+        this.ShipCity = shipCity;
+    }
+} 
+{% endhighlight %}
+
+N> If you want your data model to respond to property changes, implement `INotifyPropertyChanged` interface in your model class.
+
+Create a model repository class with OrderInfo collection property initialized with required number of data objects in a new class file as shown in the following code example and save it as OrderInfoRepository.cs file:
+
+{% highlight c# %}
+public class OrderInfoRepository
+{
+    private ObservableCollection<OrderInfo> orderInfo;
+    public ObservableCollection<OrderInfo> OrderInfoCollection {
+        get { return orderInfo; }
+        set { this.orderInfo = value; }
+    }
+
+    public OrderInfoRepository ()
+    {
+        orderInfo = new ObservableCollection<OrderInfo> ();
+        this.GenerateOrders ();
+    }
+
+    private void GenerateOrders ()
+    {
+        orderInfo.Add (new OrderInfo ("1001", "Maria Anders", "Germany", "ALFKI", "Berlin"));
+        orderInfo.Add (new OrderInfo ("1002", "Ana Trujillo", "Mexico", "ANATR", "Mexico D.F."));
+        orderInfo.Add (new OrderInfo ("1003", "Ant Fuller", "Mexico", "ANTON", "Mexico D.F."));
+        orderInfo.Add (new OrderInfo ("1004", "Thomas Hardy", "UK", "AROUT", "London"));
+        orderInfo.Add (new OrderInfo ("1005", "Tim Adams", "Sweden", "BERGS", "London"));
+        orderInfo.Add (new OrderInfo ("1006", "Hanna Moos", "Germany", "BLAUS", "Mannheim"));
+        orderInfo.Add (new OrderInfo ("1007", "Andrew Fuller", "France", "BLONP", "Strasbourg"));
+        orderInfo.Add (new OrderInfo ("1008", "Martin King", "Spain", "BOLID", "Madrid"));
+        orderInfo.Add (new OrderInfo ("1009", "Lenny Lin", "France", "BONAP", "Marsiella"));
+        orderInfo.Add (new OrderInfo ("1010", "John Carter", "Canada", "BOTTM", "Lenny Lin"));
+        orderInfo.Add (new OrderInfo ("1011", "Laura King", "UK", "AROUT", "London"));
+        orderInfo.Add (new OrderInfo ("1012", "Anne Wilson", "Germany", "BLAUS", "Mannheim"));
+        orderInfo.Add (new OrderInfo ("1013", "Martin King", "France", "BLONP", "Strasbourg"));
+        orderInfo.Add (new OrderInfo ("1014", "Gina Irene", "UK", "AROUT", "London"));
+    }
+}
+{% endhighlight %}
+
+## Binding data to the SfDataGrid
+
+To bind the data source to the SfDataGrid, set the `SfDataGrid.ItemsSource` property as follows. You can bind the data source of the SfDataGrid either from XAML or in code. 
+
+The following code example binds the collection created in previous step to `SfDataGrid.ItemsSource` property.
```

Comment:
The following code example binds the collection created in the previous step to the `SfDataGrid.ItemsSource` property: